README.md

```diff
@@ -1,17 +1,16 @@
 # Blueprint at UCI Website
 
-This is the official website for Blueprint at UCI, a student-ran organization dedicated to making technology accessible and useful for NPOs and promoting social impact in addition to giving students at UCI professional experience along the way. 
+This is the official website for Blueprint at UCI, a student-run organization dedicated to making technology accessible and useful for NPOs and promoting social impact in addition to giving students at UCI professional experience along the way. 
 
 ## Available Scripts
-
-In the project directory /website , you can run:
+In the project directory, you can run:
 
 ### `npm start`
 
-Runs the app in the development mode.<br />
```

Comment:
No need for <br/>, we can make a newline in a README by ending the line with 2 spaces :)